content/docs/guides/app_onboarding/prereqs.md

```diff
@@ -8,9 +8,12 @@ weight: 1
 
 # Prerequisites
 
-## Application UIDs
+## Security Contexts
 
-Do not run applications with the user ID (UID) value of **1500**. This is **reserved** for the Envoy proxy sidecar container injected into pods by OSM's sidecar injector.
+* Do not run applications with the user ID (UID) value of **1500**. This is **reserved** for the Envoy proxy sidecar container injected into pods by OSM's sidecar injector.
+* If security context `runAsNonRoot` is set to `true` at the pod level, a `runAsUser` value must be provided either for the pod
+or for each container. If the UID is omitted, application containers may attempt to run as root user by default, causing conflict with the pod's security context.
+* Additional capabilities are not required.
```

Comment:
OSM's init container requires the NET_ADMIN capability. This should be documented.

Reply:
added as a note since init container security contexts override user-set pod-level security contexts